scripts/jenkins-acceptance-tests.sh

```diff
@@ -37,3 +37,4 @@ pip install -r requirements/test-acceptance.txt
 
 cd test/acceptance
 python tests.py
+SELENIUM_BROWSER=phantomjs python accessibility.py
```

Comment:
So this would mean all acceptance tests go with phantom. I don't have an objection but wanted to doublecheck...